sql/core/src/test/scala/org/apache/spark/sql/execution/command/AlterTableDropPartitionSuiteBase.scala

```diff
@@ -222,7 +222,7 @@ trait AlterTableDropPartitionSuiteBase extends QueryTest with DDLCommandTestUtil
       }
 
       val v2 = s"${spark.sharedState.globalTempViewManager.database}.v2"
-      withGlobalTempView(v2) {
+      withGlobalTempView("v2") {
```

Comment:
This is a bug fix. `withGlobalTempView()` doesn't remove global temp view if it is called with full path: `withGlobalTempView("global_temp.v2")`